lib/ModelMixins/TableMixin.ts

```diff
@@ -149,6 +151,83 @@ export default function TableMixin<T extends Constructor<Model<TableTraits>>>(
       ]);
     }
 
+    /**
+     * Gets the items to show on a chart.
+     *
+     * TODO: rename to chartItems() when we have removed all references to the
+     * current implementation.
```

Comment:
What's the plan for returning everything to use `chartItems` again?

Reply:
The current `chartItems()` method is called from the old chart stack (mostly `ChartRenderer`) which will become redundant once we merge this PR.  Once we are ok to merge this PR, I'll remove `ChartRenderer` and  friends and  rename `chartItems2 -> chartItems`.